src/pages/components/date-picker/usage.mdx

```diff
@@ -565,6 +575,67 @@ a meeting time.
 </Column>
 </Row>
 
+## AI presence
+
+Date picker has a modification that takes on the AI visual styling when the AI
+slug is present in the input. The AI variant functions the same as the normal
+version except with the addition of the AI slug which is both a visual indicator
+and the trigger for the explainability popover.
+
+More information about designing for AI guidelines is coming soon.
+
+<Row>
+<Column colLg={8}>
+
+<Tabs>
+
+<Tab label="Default">
+
+![Enabled state of default date picker with AI presence](images/date-picker-usage-AI-presence-default.png)
+
+</Tab>
+
+<Tab label="Fluid">
+
+![Enabled state of fluid date picker with AI presence](images/date-picker-usage-AI-presence-fluid.png)
+
+</Tab>
+
+</Tabs>
+
+</Column>
+</Row>
+
+### Revert to AI
+
+A date picker can toggle between the AI variant and the non-AI variant depending
+on the user’s interaction. If the user manually overrides the AI-suggested
+content then the input will change from the AI variant to the default variant.
```

Comment:
Hi @Kritvi-bhatia17 just noticed this default to non-AI variant change in the usage tab for Date picker and Select component, rest is all good! 🚀 

_"content then the input will change from the AI variant to the **non-AI** variant."_